task/s2i-dotnet/0.1/s2i-dotnet.yaml

```diff
@@ -28,6 +28,10 @@ spec:
       description: The location of the path to run s2i from.
       default: .
       type: string
+    - name: STARTUP_PROJECT
+      description: The location of the .NET project file or its parent folder.
+      default: .
```

Comment:
This isn't a change from the current functionality, is it? In other words, if users didn't specify `STARTUP_PROJECT` is there any case this will result in a different behaviour compared to before this change>

Reply:
Yes, this is backwards compatible. The default used here matches the default the S2I builder has for this variable.